packages/json-file/package.json

```diff
@@ -34,7 +34,7 @@
     "write-file-atomic": "^2.3.0"
   },
   "devDependencies": {
-    "@expo/babel-preset-cli": "0.2.9",
```

Comment:
I think we'll still want to have this dependency, `tsconfig.json` extends the config found in it:
```json
"extends": "@expo/babel-preset-cli/tsconfig.base",
```
(Even though the package is a bit poorly named, since it does not only include a babel preset. I'll soon work on the tsc setup, so no need to change that now.)

Reply:
Ok, I'll revert it! 😄